### ISSUE TITLE
Get the confidence value for the classified output

### ISSUE THREAD

**Issue by Sreeni1204** [NVIDIA/NeMo#7043]
Hello,

I am trying to get the confidence value with which the output labels are generated. Like for example, given a *.wav as input, I am able to extract some label from the model. But I would like to know the confidence level with which the labels are extracted for a single audio. I would need this information for some kind of thresholding in my application. 

Please suggest some ideas on how to get that information. Also please do comment if you need more information.

**Comment by nithinraok** [comment]
argmax on logits should give you what you are looking for. 
eg: `logits.argmax(axis=1)`
